client/asset/btc/livetest/livetest.go

```diff
@@ -118,6 +118,8 @@ func (rig *testRig) mineAlpha() error {
 	tmuxWindow := rig.symbol + "-harness:2"
 	if rig.symbol == "zec" {
 		tmuxWindow = rig.symbol + "-harness:4"
+	} else if rig.symbol == "firo" {
+		tmuxWindow = rig.symbol + "-harness:4"
 	}
```

Comment:
Looks like "doge" is also a window-4 tmux setup that we missed.  Let's convert this to a `switch` as follows;

```go
var tmuxWindow string
switch rig.symbol {
case "zec", "firo", "doge":
	tmuxWindow = rig.symbol + "-harness:4"
default:
	tmuxWindow = rig.symbol + "-harness:2"
}
```

Reply:
Better and future-proof, thanks
Updated and will push